core-library/src/main/java/org/silverpeas/core/notification/user/client/UserNotification.java

```diff
@@ -28,26 +28,21 @@
 import org.silverpeas.core.util.URLUtil;
 import org.silverpeas.core.util.logging.SilverLogger;
 
-/**
- * Class declaration
- * @author
- * @version %I%, %G%
- */
-public abstract class AbstractNotification {
+public interface UserNotification {
```

Comment:
UserNotification is an interface which is already existing (in the parent package according to my memory).
Is it possible to modify the renaming from AbstractNotification to UserNotification by AbstractNotification to NotificationUrlProvider (or something in the mind)?

Reply:
ok